modules/combined/test/tests/thermal_elastic/tests

```diff
@@ -1,17 +1,25 @@
 [Tests]
+  design = 'ThermalConductivity.md'
+  issues = '#13232'
   [./test]
     type = 'Exodiff'
     input = 'thermal_elastic.i'
     exodiff = 'thermal_elastic_out.e'
+    requirement = "The system shall compute constant stress in the xx, yy, zz, xy, yz, and xz direction
+                   using irregular elements with functional Poisson's ratios and Young's moduli."
   [../]
   [./test_sm]
     type = 'Exodiff'
     input = 'thermal_elastic_sm.i'
     exodiff = 'thermal_elastic_out.e'
     prereq = test
+    requirement = "The system shall compute the same stress values as the thermal_elastic.i test using the
+                   solid mechanics module implementation  using the solid mechanics module."
   [../]
   [./derivatives]
     type = 'AnalyzeJacobian'
+    petsc_version = '<3.9.0 || >=3.9.4'
```

Comment:
FYI @fdkong @permcody This type of logical checking doens't work on my MacOS High Sierra:
`[using PETSc 3.9.4 REQ: < 3.9.0] SKIP`

Reply:
Thanks for the report. I will check